irohad/main/application.hpp

```diff
@@ -14,9 +14,11 @@
 #include "interfaces/queries/query.hpp"
 #include "logger/logger_fwd.hpp"
 #include "logger/logger_manager_fwd.hpp"
+#include "main/server_runner.hpp"
```

Comment:
why did you change the forward declaration for header inclusion?

Reply:
`boost::optional<std::shared_ptr<ServerRunner>>` seems to require a full type, otherwise my compiler gives `error: invalid application of 'sizeof' to an incomplete type 'ServerRunner'` (clang-10.0.1 on macos)

Reply:
Tested it. Seems you are right, and I am not. Then this include is OK.